src/components/common/PeopleAndRoles/ListPeopleAndRoles.vue

```diff
@@ -283,7 +283,7 @@
 
               <!-- org-person we haven't touched: -->
               <div v-else class="actions mr-4">
-                <span class="edit-action">
+                <span class="edit-action" v-if="showSoleProprietorChangeButton">
```

Comment:
As on line 299 and 300, what do you think about labelling this "canEdit" ?

I'm thinking "separation of concerns" -- this template doesn't need to know about SP or Proprietor or that the edit label is "Change". KISS here, and in the script you can explain the reason for the logic.
This works well with moving the getter from the store to this component (or its mixin) -- the store doesn't need to ko

Reply:
Let's hold off deciding on an appropriate name until we've decided on the appropriate location -- getter vs component.

Reply:
Hehehe